src/applications/vaos/components/review/Description.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { getTypeOfCare } from '../../utils/selectors';
 import { FLOW_TYPES, FACILITY_TYPES } from '../../utils/constants';
-import { lowerCase } from '../../utils/appointment';
+import { lowerCase } from '../../utils/formatters';
```

Comment:
`lowerCase()` isn't used anywhere else.  Can we move it from formatters into here?